src/main/resources/db/changelog/db.changelog-RDM-10167.xml

```diff
@@ -0,0 +1,35 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+<changeSet id="RDM-10167" author="tobias.derose@hmcts.net">
+    <createTable tableName="message_queue_candidates">
+
+        <column name="id" type="bigint">
+            <constraints nullable="false" primaryKey="true"/>
```

Comment:
mmm not completely sure Tobias but do we need to declare this with autoincrement?